code/modules/research/data-analysis/decryption.dm

```diff
@@ -0,0 +1,171 @@
+/obj/machinery/computer/decryption
+	name="Data-Decryption Console"
+	desc="This console is used to analyse and decrypt data found from various sources"
+	icon = 'icons/obj/computer.dmi'
+	icon_screen = "dna"
+	icon_keyboard = "med_key"
+	density = 1
+	var/obj/item/weapon/book/databook/databook=null
+	// Currently loaded databook.
+	var/init = 1
+
+	var/A
```

Comment:
Write it out to alpha fragment, beta fragment etc? Or add some comment lines to clarify?